src/lib/context-menu.ts

```diff
@@ -305,24 +312,31 @@ export class ContextMenu {
           this.createMenuSection((section: HTMLElement) => {
             section.appendChild(
               this.createMenuItem('Add Reference Note', () => {
-                const footnotesElement = findChildrenByType(
+                const footnotesElementWithPos = findChildrenByType(
                   this.node,
                   schema.nodes.footnotes_element
-                ).pop()?.node
```

Comment:
Same as above, this is meant to be `pop()`.

Reply:
reverted